Resources/config/service.xml

```diff
@@ -1,24 +1,17 @@
 <?xml version="1.0" ?>
-<container xmlns="http://symfony.com/schema/dic/services"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
-
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
         <parameter key="salva_assetic_filter.jshrink.class">Salva\JshrinkBundle\Assetic\Filter\JshrinkFilter</parameter>
         <parameter key="salva_twig_extension.jshrink.class">Salva\JshrinkBundle\Twig\Extension\JshrinkExtension</parameter>
     </parameters>
-
     <services>
-
-        <service id="salva_assetic_filter.jshrink" class="%salva_assetic_filter.jshrink.class%">
-            <argument />
-            <tag name="assetic.filter" alias="jshrink"></tag>
+        <service class="%salva_assetic_filter.jshrink.class%" id="salva_assetic_filter.jshrink">
+            <argument>null</argument>
```

Comment:
Using `<argument />` an empty string is passed to those classes, while using `<argument>null</argument>` the `NULL` value is passed (and so the default argument is used).

I tested it here: http://converter.rosstuck.com
Maybe it's the same...